### ISSUE TITLE
Improve project structure image.

### ISSUE THREAD

**Issue by ClientCrash** [cryptic-game/cryptic#63]
**Describe the bug**
Black on dark blue is hard to read. I also recommend switching to a more "serious" font.

**Comment by cadamini** [comment]
This is not a bug for me. I agree that it could be enhanced but it doesn't sound as the top priority.

**Comment by DevMimas** [comment]
@ClientCrash Yes, that's right. The image needs to be completely replaced anyway. It will be done soon.

**Comment by Webkinger** [comment]
Beispiel:
![Projekt struktur](https://user-images.githubusercontent.com/33805308/162584063-45a9eb61-7a97-4e67-9e30-622fb51bc1cb.jpg)

**Comment by ClientCrash** [comment]
Ich wäre dafür das erstmal zu übernehmen bis eine entgültige Lösung gefunden ist. Das hier ist schon einmal eine deutliche verbesserung zum aktuellen Zustand.
